concent_api/core/queue_operations.py

```diff
@@ -3,7 +3,8 @@
 from conductor.tasks import blender_verification_request
 from common.helpers import get_storage_result_file_path
 from common.helpers import get_storage_source_file_path
-from core.utils import extract_name_from_scene_file_path
+from core.utils import extract_name_from_scene_file_path, adjust_format_name
```

Comment:
Please review imports in this pull request.

Reply:
fixed